File: App/app/src/main/java/com/cinema/ticket_booking/data/model/request/ReviewRequest.java
package com.cinema.ticket_booking.data.model.request;
public class ReviewRequest {
    private String movieId, bookingId, comment;
    private int rating;
    public ReviewRequest(String movieId, String bookingId, int rating, String comment) {
        this.movieId = movieId; this.bookingId = bookingId;
        this.rating = rating; this.comment = comment;
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/request/ReviewRequest.kt
package com.cinema.ticket_booking.data.model.request

import com.google.gson.annotations.SerializedName

data class ReviewRequest(
    @SerializedName("movieId")
    val movieId: String?,
    @SerializedName("bookingId")
    val bookingId: String?,
    @SerializedName("rating")
    val rating: Int,
    @SerializedName("comment")
    val comment: String?
)
